### PR TITLE
[contacts][stories] add stories for expo-contacts

### PR DESCRIPTION
**Pull request by ajsmth**
# Why

This is an example of what more targeted stories / components might look like for a given SDK module

https://user-images.githubusercontent.com/40680668/132395634-509b791b-d7d3-4a1d-afdf-0407a6b7eff0.mov

Adding more examples that target specific parts of an API should be straightforward by adding them to the list: 
Screen Shot 2021-08-06 at 3 27 02 PM

# Testing

- navigate to `expo-contacts` and run `yarn start:examples` - note only iOS is done right now for custom builds

Example of test suite in use: 

https://user-images.githubusercontent.com/40680668/128581302-ce9f7477-dcd9-4148-a50d-15084e344bc0.mov

### CODE REVIEW

File: packages/expo-contacts/src/__stories__/components/AddContact.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import * as Contacts from 'expo-contacts';
+import { Button } from 'expo-stories/shared/components';
+import * as React from 'react';
+
+import { useContact } from '../helpers';
+import { ContactCard } from './ContactCard';
+
+type IAddContact = {
```
Comment (bbarthec) [inline packages/expo-contacts/src/__stories__/components/AddContact.tsx:8]: I'm not a big fun of these `I<TypeName>` prefixes 🤔 Why not to follow the idea of `AddContactProps` (mimicking the RN's types?)

**Review by bbarthec — COMMENTED** [review]
This is nice 🥳 
There's almost nothing that I can complain about 💪 
My remarks:
- we don't practice using `I<Component/Type>` prefixes - name them semantically (like `ComponentProps`, etc)
- components names should rather be nouns than verbs/actions names (`IsAvailable` => `IsAvailbleCard`, `UpdateContant` => `UpdateContactView`, etc)

File: packages/expo-contacts/package.json [hunk 2]
```diff
@@ -41,5 +42,10 @@
   "devDependencies": {
     "@types/uuid": "^3.4.7",
     "expo-module-scripts": "^2.0.0"
+  },
+  "expoStories": {
+    "packages": {
+      "expo-contacts": "~9.2.0"
+    }
```
[the remark this answers is not in the document]
Comment (ajsmth) [reply packages/expo-contacts/package.json:49]: im inclined to agree with you on this - I will trial it and see what happens, although I'm hesitant to remove it all together until more examples have been completed as I am not sure what kind of escape hatches we will need

File: packages/expo-contacts/src/__stories__/components/AddContact.tsx (hunk 1 above)
Comment (ajsmth) [reply packages/expo-contacts/src/__stories__/components/AddContact.tsx:8]: this is an old habit of mine - sorry!

**Review by bbarthec — APPROVED** [review]

**Review by EvanBacon — APPROVED** [review]
just ensure no extra files are being added to the npm package (`npm pack`)

**Pull request closed without merging**